entrypoint-test.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
 set -e
 cd /app
-./manage.py create_citext_extension --database template1
-./manage.py test --noinput --failfast
+uv run ./manage.py create_citext_extension --database template1
```

Comment:
Do we still need to create the citext extension? That should have been resolved in https://github.com/unioslo/mreg/pull/513

Reply:
Probably a regression from a bad rebasing job by me.

Reply:
Nevermind, that is still present in master. We can remove that here or in another PR.